tests/components/honeywell/test_climate.py

```diff
@@ -1207,3 +1209,66 @@ async def test_unique_id(
     await init_integration(hass, config_entry)
     entity_entry = entity_registry.async_get(f"climate.{device.name}")
     assert entity_entry.unique_id == str(device.deviceid)
+
+
+async def test_preset_mode(
+    hass: HomeAssistant,
+    device: MagicMock,
+    config_entry: er.EntityRegistry,
+) -> None:
+    """Test mode settings properly reflected."""
+    await init_integration(hass, config_entry)
+    entity_id = f"climate.{device.name}"
+
+    device.raw_ui_data["StatusHeat"] = 3
+    device.raw_ui_data["StatusCool"] = 3
+
+    async_fire_time_changed(
+        hass,
+        utcnow() + SCAN_INTERVAL,
+    )
```

Comment:
can we use the freezer?

Reply:
Freezer?  I'm not familiar with that.  I need the device to be polled (via preset mode) to trigger the code that "sees" the hold mode, then when I turn off the "hold mode" remotely, see that it left the away mode....

Reply:
Oh I mean the 
```py
    freezer.tick(SCAN_INTERVAL)
    async_fire_time_changed(hass)
    await hass.async_block_till_done()
```

Reply:
Why yes we can.  Now this one is different from the rest of the tests in this file... but it does work.